tests/QARegressionTests/test_Issues/test_CoreIssue576.py

```diff
@@ -22,48 +31,36 @@ def test_negative_dataset():
     stdout, stderr = process.communicate()
 
     file_name = stdout.decode().strip().split(": ")[1] + ".xlsx"
-    print(file_name)
     # Open the Excel file
     workbook = openpyxl.load_workbook(file_name)
 
     # Go to the "Issue Details" sheet
-    sheet = workbook["Issue Details"]
+    Issues_Sheet = workbook["Issue Details"]
+    Dataset_Sheet = workbook["Dataset Details"]
+    Rules_Sheet = workbook["Rules Report"]
```

Comment:
These workbook labels should come from globally-defined constants or enum within the reg tests. Otherwise, if we happen to change a label it will break all of the tests.

Reply:
updated